### DISCUSSION TITLE
usecase.json & parameters.json via http

### DISCUSSION THREAD

**Discussion by AnatolManikalo** [SAP-archive/btp-setup-automator#395] [Q&A] [answered]
Hello colleagues,

If I understood the diagram here: https://github.com/SAP-samples/btp-setup-automator/blob/main/docs/OVERVIEW.md correctly, there should be a possibility to store usecase.json & parameters.json files somewhere (e.g. in GitHub) and use them via https(s) instead of using them locally as part of the package. 

Could you please share how to achieve this?

Thanks,
Anatol

**Comment by lechnerc77** [comment, marked as answer]
Hi @AnatolManikalo, 

you can find an example [here](https://github.com/SAP-samples/btp-setup-automator/blob/main/docs/README.md#step-4-run-the-btp-setup-automator). The examples uses `docker exec` but it is the same when you use the `btpsa` inside the container. Be aware to reference the raw file content.

If you need to authenticate to access the file content, this [issue](https://github.com/SAP-samples/btp-setup-automator/issues/310#issuecomment-1337528879) shows you how to do that. This functionality is available since release [1.2.0](https://github.com/SAP-samples/btp-setup-automator/releases/tag/btpsa-v1.2.0).

Best,
Christian

**Comment by AnatolManikalo** [comment]
Hi @lechnerc77 

Thanks for your reply. If we want to use local parameters file (not http), should this file be inside built container, or is it possible to use one outside container? How to provide path to it then?

**Reply by lechnerc77** [reply]
This could be achieved via mounting the local filesystem to the Docker container. Please check the Docker documentation on how to achieve the mounting to the container.
